commands/root.go

```diff
@@ -143,6 +144,7 @@ func WelcomeAction(c *console.Context) error {
 		localServerStopCmd,
 		localSecurityCheckCmd,
 		composerWrapper,
+		pieWrapper,
```

Comment:
I don't think it deserves to be in this list, which should be kept as short as possible.

Reply:
This means that it won't be shown in the available wrappers when running `symfony -h`, I'm afraid it makes it undiscoverable?

Reply:
Removed from this list and added to the list of available wrappers when using `-h`